### PR TITLE
Makes ticket price clearer

### LINKED ISSUE #916: Make ticket price clearer in governance pages
A user on Twitter recently pointed out his [confusion](https://twitter.com/reganbozman/status/1113996236917440518) that one needs to have enough DCR to purchase a ticket. Upon reading the docs with fresh eyes, it does seem we should make this clearer.

### PR DESCRIPTION
**Pull request by s-ben**
This PR updates the 'Proof-of-Stake (PoS) Voting' and 'Introduction to Decred Governance' pages to include text that makes it clear there a certain number of DCR needs to be time-locked to participate in voting. Our current wording simply says you need to time-lock DCR, and goes into what that means, without mentioning ticket price until way down the page, in sections going into more technical details ( Closes #916 ).

### CODE REVIEW

File: docs/proof-of-stake/overview.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ These roles are incentivized; ticket-holders (or "Proof-of-Stake (PoS) Voters")
...
+To participate in PoS voting, stakeholders lock some DCR in return for a ticket. The amount of DCR locked, or **Ticket Price**, is adjusted dynamically every 144 blocks (~12 hrs). The current ticket price can be found in [Decrediton](../wallets/decrediton/decrediton-setup.md) or on [dcrstats.com](https://dcrstats.com). Partial tickets can also be purchased using [ticket splitting](ticket-splitting.md) software. Every ticket owned gives its holder the ability to cast a single vote. Upon voting, each ticket returns a small reward plus the original **Ticket Price** of the ticket. Each ticket is selected to vote at random, giving an average vote time of 28 days, but possibly requiring up to 142 days, with a 0.5% chance of expiring before being chosen to vote (this expiration returns the original **Ticket Price** without a reward). Every block mined must include a minimum of 3 votes (miners are penalized by a reward deduction if fewer than 5 votes are included).
```
Comment (dajohi) [inline docs/proof-of-stake/overview.md]: change dcrstats.com to explorer.dcrdata.org please.

File: docs/governance/introduction-to-decred-governance.md [hunk 2]
```diff
@@ -8,7 +8,8 @@ Decred's governance is based on the principle of ticket-holder voting. The ultim
 
 ---
 
-Holders of DCR can time-lock their funds in exchange for [tickets](../proof-of-stake/overview.md). Tickets allow one to participate in Decred's governance in three ways, two on-chain and one off-chain.
+Holders of DCR can time-lock their funds in exchange for [tickets](../proof-of-stake/overview.md). The ticket price (the amount that must be time-locked) is adjusted dynamically every 144 blocks (~12 hrs), in an attempt to in order to keep PoS subsidy returns stable over time. Tickets allow one to participate in Decred's governance in three ways, two on-chain and one off-chain.
```
Comment (jholdstock) [inline docs/governance/introduction-to-decred-governance.md]: grammar: `in an attempt to in order to`.

Comment (s-ben) [reply docs/governance/introduction-to-decred-governance.md]: Good catch. Went with `in an attempt to`.

File: docs/proof-of-stake/overview.md (hunk 1 above)
Comment (s-ben) [reply docs/proof-of-stake/overview.md]: Done.

**Pull request merged**